src/tools/mesh_tester.cpp

```diff
@@ -250,6 +328,109 @@ std::vector< std::pair<int, int> > naiveIntersectionAlgorithm(
   return retval;
 }
 
+#if defined(AXOM_USE_RAJA)
+template < typename ExecSpace >
+std::vector< std::pair<int, int> > naiveIntersectionAlgorithm(
+  mint::Mesh* surface_mesh,
+  std::vector<int> & degenerate)
+{
+  SLIC_INFO("Running naive intersection algorithm "
+    << " in execution Space: " 
+    << axom::execution_space< ExecSpace >::name());
+
+  // Get allocator
+  #ifdef AXOM_USE_UMPIRE
+    int allocatorID = axom::execution_space< ExecSpace >::allocatorID();
+    umpire::Allocator allocator = axom::getAllocator(allocatorID);
+  #endif
+
+  std::vector< std::pair<int, int> > retval;
+
+  const int ncells = surface_mesh->getNumberOfCells();
+  SLIC_INFO("Checking mesh with a total of "<< ncells<< " cells.");
+
+  #ifdef AXOM_USE_UMPIRE
```

Comment:
@gzagaris -- This is what I mentioned the other day.  We should update the ``axom::allocate`` interface to allow a dummy (integer) allocator ID when Umpire is not available.

See: 
https://github.com/LLNL/axom/blob/146f2b12fa1ffe4d430a449e13adece30d1a5de2/src/axom/core/memory_management.hpp#L98-L105

Reply:
Yes -- I'll take care of this. Again, we should just take an integer ID and don't expose the `umpire` objects in the API.

Reply:
Fixed in 0ed1af3682b6fdf71bc08c4788f94aa0d54993f1